pkg/alertmanager/alertstore/store.go

```diff
@@ -38,6 +38,14 @@ type AlertStore interface {
 	// If configuration for the user doesn't exist, no error is reported.
 	DeleteAlertConfig(ctx context.Context, user string) error
 
+	GetGrafanaAlertConfig(ctx context.Context, user string) (alertspb.GrafanaAlertConfigDesc, error)
+	SetGrafanaAlertConfig(ctx context.Context, cfg alertspb.GrafanaAlertConfigDesc) error
+	DeleteGrafanaAlertConfig(ctx context.Context, user string) error
+
+	GetFullGrafanaState(ctx context.Context, user string) (alertspb.FullStateDesc, error)
+	SetFullGrafanaState(ctx context.Context, user string, fs alertspb.FullStateDesc) error
+	DeleteFullGrafanaState(ctx context.Context, user string) error
+
```

Comment:
nit: the other Delete* methods have a doc string saying how they behave if the user doesn't exist or doesn't have saved config. Can you add a line with those too?

Reply:
Solved in [083abea](https://github.com/grafana/mimir/pull/6682/commits/083abea0147e15612272e7bfab850550f649e3db)